org.alloytools.alloy.application/src/test/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslationTest.java

```diff
@@ -89,12 +89,9 @@ public void testStateInitAndUnnamedRelations() throws Exception {
             assertTrue(decl.contains("="));
         }
 
-        // Unamed relation should be generated.
-        assertTrue(translation.relations.contains(new DashPythonTranslation.Relation("Chair_Player", "[Chair, Player]")));
-
         assertTrue(gameState.getDecls().contains("Game_active_players = Player('active_players', 3)"));
         assertTrue(gameState.getDecls().contains("Game_active_chairs = Chair('active_chairs', 3)"));
-        assertTrue(gameState.getDecls().contains("Game_occupied = Chair_Player()"));
+        assertTrue(gameState.getDecls().contains("Game_occupied = Chair * Player"));
```

Comment:
What is the type of a cartesian product, is it a relation?

Reply:
It is of type "set".